File: sistemastreamingjs/pelicula.js
class Pelicula {
    constructor(titulo = "PeliculaDefault", director = "DirectorDefault", duracion = 0) {
        this.titulo = titulo;
        this.director = director;
        this.duracion = duracion;
    }

    toString() {
        return `Título: ${this.titulo}, Director: ${this.director}, Duración: ${this.duracion} min`;
    }
}

module.exports = Pelicula;

File: sistemastreamingruby/pelicula.rb
class Pelicula
    attr_accessor :titulo, :director, :duracion
  
    def initialize(titulo = "PeliculaDefault", director = "DirectorDefault", duracion = 0)
      @titulo = titulo
      @director = director
      @duracion = duracion
    end
  
    def to_s
      "Título: #{@titulo}, Director: #{@director}, Duración: #{@duracion} min"
    end
  end
  